### ISSUE TITLE
Strong name signature could not be verified

### ISSUE THREAD

**Issue by ebugusey** [efcore/EFCore.NamingConventions#23]
.NET throws this error, when I use this library in "good" old WCF.

```
Could not load file or assembly 'EFCore.NamingConventions' or one of its dependencies. 
Strong name signature could not be verified. The assembly may have been tampered with,
or it was delay signed but not fully signed with the correct private key.
(Exception from HRESULT: 0x80131045)
```
I'm using .NET Framework 4.7.1 (it's compatible with netstandard2.0) and version 1.1.0 of this library.

**Comment by alexvab** [comment]
We also have this issue in the asp.net mvc 5.x app 
```
Could not load file or assembly 'EFCore.NamingConventions' or one of its dependencies. Strong 
name signature could not be verified.  The assembly may have been tampered with, or it was delay 
signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)
```
.NET Framework 4.8 and version 1.1.0 of this library.

**Comment by austindrenski** [comment]
Hey @roji  👋 

Is there anything blocking this from being backported + published to nuget?

**Comment by roji** [comment]
@austindrenski sorry, I somehow missed this message... Is this still relevant, given that version 5.0 is now out? If so I can probably release a 1.1.0 patch for this.

**Comment by aldobrynin** [comment]
@roji yes, please. We'd appreciate that

**Comment by roji** [comment]
Have just published [1.1.1](https://www.nuget.org/packages/EFCore.NamingConventions/1.1.1) which disables public signing.
